proposals/release-approval-process.md

```diff
@@ -8,9 +8,9 @@ This approval process hopes to encourage early consistent consensus building dur
 
 **Planning a release:** Every OCI specification project SHOULD hold a weekly meeting that involves maintainers reviewing pull requests, debating outstanding issues, and planning releases. This meeting MUST be advertised on the project README and MAY happen on a phone call, video conference, or on IRC. Maintainers MUST send updates to the dev@opencontainers.org with results of these meetings. Maintainers MAY change the meeting cadence once a specification has reached v1.0.0. The meeting cadence MUST NOT be greater than once every four weeks The release plans, corresponding milestones and estimated due dates MUST be published on GitHub (e.g. https://github.com/opencontainers/runtime-spec/milestones). GitHub milestones and issues are only used for community organization and all releases MUST follow this process.
 
-**Making a release:** OCI specification projects MUST announce intentions to release with two project maintainer sponsors (listed in the repo MAINTAINERS file) on the dev@opencontainers.org mailing list. After the announcement a two-thirds super majority of project maintainers MUST reply to the list with an LGTM within one week for the release to be approved. The maintainers MUST wait a full week for maintainers to reply but if all maintainers reply with an LGTM then the release MAY release earlier except in the case of a major releases.
+**Making a release:** OCI specification projects MUST announce intentions to release with two project maintainer sponsors (listed in the repo MAINTAINERS file) on the dev@opencontainers.org mailing list. After the announcement two-thirds of the voting project maintainers MUST reply to the list with an LGTM within one week for the release to be approved. The maintainers MUST wait a full week for maintainers to reply but if all maintainers reply with an LGTM then the release MAY release earlier except in the case of a major releases.
```

Comment:
This still [doesn't have quorum language][1].  And [I do like Robert's “two-thirds of the votes cast, a quorum being present”][2].  How about:

> Voting on proposed releases should happen on the appropriate mailing list with maintainers posting LGTM or REJECT.  They may also explicitly not vote by posting ABSTAIN (which is useful to revert a previous vote).  Maintainers may post multiple times (e.g. as they revise their position based on feeback), but only their final post counts as their vote.  A proposed release passes if two-thirds of votes cast, a quorum having voted, are in favor of the release.  A quorum is established when at two-thirds of maintainers have voted.  Voting SHOULD remain open for a week, although under exceptional conditions (e.g. security fixes) non-major releases which reach quorum with unanimous support MAY be released earlier.

[1]: https://github.com/opencontainers/tob/pull/15#discussion_r67037985
[2]:https://github.com/opencontainers/tob/pull/3#issuecomment-195066430